.travis.yml

```diff
@@ -2,20 +2,31 @@ language: go
 go:
   - 1.10.2
 
-# Don't want default ./... here:
-install:
-- export PATH=$GOPATH/bin:$PATH
-- mkdir -p $HOME/gopath/src/k8s.io
-- mv $TRAVIS_BUILD_DIR $HOME/gopath/src/k8s.io/website && cd $HOME/gopath/src/k8s.io/website
+env:
+  - HUGO_VERSION=0.49
 
-# Make sure we are testing against the correct branch
-- pushd $GOPATH/src/k8s.io && git clone https://github.com/kubernetes/kubernetes && popd
-- pushd $GOPATH/src/k8s.io/kubernetes && git checkout release-1.11 && popd
-- cp -L -R $GOPATH/src/k8s.io/kubernetes/vendor/ $GOPATH/src/
-- rm -r $GOPATH/src/k8s.io/kubernetes/vendor/
+jobs:
+  include:
+    # Don't want default ./... here:
+    - install:
+      - export PATH=$GOPATH/bin:$PATH
+      - mkdir -p $HOME/gopath/src/k8s.io
+      - mv $TRAVIS_BUILD_DIR $HOME/gopath/src/k8s.io/website && cd $HOME/gopath/src/k8s.io/website
 
-# Fetch additional dependencies to run the tests in examples/examples_test.go
-- go get -t -v k8s.io/website/content/en/examples
+      # Make sure we are testing against the correct branch
+      - pushd $GOPATH/src/k8s.io && git clone https://github.com/kubernetes/kubernetes && popd
+      - pushd $GOPATH/src/k8s.io/kubernetes && git checkout release-1.11 && popd
+      - cp -L -R $GOPATH/src/k8s.io/kubernetes/vendor/ $GOPATH/src/
+      - rm -r $GOPATH/src/k8s.io/kubernetes/vendor/
 
-script:
-- go test -v k8s.io/website/content/en/examples
+      # Fetch additional dependencies to run the tests in examples/examples_test.go
+      - go get -t -v k8s.io/website/content/en/examples
+      script:
+      - go test -v k8s.io/website/content/en/examples
+    - install:
```

Comment:
IIUC, line 26-32 are new commands to be executed. Basically we are adding a new 'install' and a new 'script' entry. I'm not sure there is a better (more intuitive) organization of these entries.

Reply:
@tengqm Thanks. I don't think there is a better way to organize these entries because two of them are quite different each other for `go test` and `hugo`.

Reply:
@tnir Just check travis documentation [here](https://docs.travis-ci.com/user/build-stages/) where there are concepts of "stages". Maybe we can use it for better organization? By "better organization", I mean more explicitly grouping the commands into "stages", e.g. "testing examples", "hugo tests" etc.

Reply:
@tengqm Currently we are getting the following jobs with name (cf. [kubernetes/website Build #441445585](https://travis-ci.org/kubernetes/website/builds/441445585)) but we should name each build?:

```
Build Jobs
 #25327.1       Go: 1.10.2       HUGO_VERSION=0.49       2 min 32 sec
 #25327.2       Go: 1.10.2       HUGO_VERSION=0.49             46 sec
```

Named builds are available at https://travis-ci.org/kubernetes/website/builds/441506030

Reply:
Cool, looks like travis is capable of naming builds automatically. If we, for example, add a 'stage' key to each job, maybe Travis will pick it up? Just some random thoughts on improving the PR.

Reply:
Yes, Travis CI does, but at the moment I don't think we do have to do that at all as there isn't any other "stage".